pkgs/development/compilers/bs-platform/build-bs-platform.nix

```diff
@@ -35,11 +35,12 @@ stdenv.mkDerivation {
   '';
 
   buildPhase = ''
-    node scripts/ninja.js build
+    # This is an unfortunate name, but it's actually how to build a release
+    # binary for BuckleScript
+    node scripts/install.js
   '';
 
   installPhase = ''
-    node scripts/install.js
```

Comment:
ah I see so scripts/install.js is esentially not needed for production build.

Reply:
`scripts/ninja.js build` is not needed for a production build. `scripts/install.js` is

Reply:
yes that's what I meant, sorry for confusion. I should stop doing review early in the morning :D